Makefile

```diff
@@ -209,6 +209,22 @@ node: curl gpg
 		sudo apt-get install -qqy nodejs; \
 	fi
 
+gofmt:
+	if [[ "$$(which gofmt)" != "$(GOPATH)/bin/gofmt" ]]; then \
+		TMP_DIR=$$(mktemp -d); \
+		pushd $$TMP_DIR > /dev/null 2>&1; \
+		git clone "https://github.com/golang/go.git"; \
+		cd go; \
+		git checkout "origin/release-branch.go1.11"; \
+		mv "src/cmd" "$(GOPATH)/src/cmd"; \
+		popd > /dev/null 2>&1; \
+		rm -rf "$$TMP_DIR"; \
+		pushd "$(GOPATH)/src" > /dev/null 2>&1; \
```

Comment:
We can simply `cd` here. The paired `popd` happens at the very end, at which point we don't care about CWD any more (recipes run in a subshell so we don't need to restore CWD at the end of a recipe).